test/integration/targets/gitlab_project_variable/aliases

```diff
@@ -0,0 +1,3 @@
+shippable/posix/group1
+gitlab/ci
+disabled
```

Comment:
Ah, one final question: why are the tests disabled? If the tests can't be run via CI, this should be `unsupported`; `disabled` is for tests which fail and are temporarily disabled until they are fixed.

Reply:
It's the same here. I followed the other gitlab modules and they have the same content in `aliases`.  
I guess they are disabled because no one provided credentials for the CI to test?